pypeit/edgetrace.py

```diff
@@ -4295,14 +4359,14 @@ def _fill_design_table(self, ind, coeff, omodel_bspat, omodel_tspat):
         self.design['ALIGN'] = self.spectrograph.slitmask.alignment_slit[ind].astype(
                                         dtype=self.design['ALIGN'].dtype)
 
-    def _fill_objects_table(self, ind):
+    def _fill_objects_table(self, maskdef_id):
```

Comment:
Same comment as above.  Please define precisely the Table columns.

I should ask the same in one of the `.rst` files.  Thanks.

Reply:
Note:  I am adding `OBJNAME` in the `deimos_radec` branch.

Reply:
Done the doc string. I will update this to include `OBJNAME` once I move to the `deimos_radec` branch.

I am not sure which `.rst` file I should update. Can you point it out to me?

Reply:
If the items are specific to DEIMOS then use `deimos.rst`  
If not, then I thknk `slit_tracing.rst` would be good.
Thanks

Reply:
Ok! I decided to add a doc for MasterSlits that was missing, and updates some other docs. Please let me know if you think everything is alright.